drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/editor/BaseGuidedDecisionTableEditorPresenterTest.java

```diff
@@ -76,11 +78,15 @@
 
     @Mock
     private SaveAndRenameCommandBuilder<GuidedDecisionTable52, Metadata> saveAndRenameCommandBuilder;
+    @Mock
+    private AuthoringWorkbenchDocks docks;
 
     @Override
     protected GuidedDecisionTableEditorPresenter getPresenter() {
         return new GuidedDecisionTableEditorPresenter(view,
                                                       dtServiceCaller,
+                                                      docks,
```

Comment:
Either use `@Mock` or `mock(..)` unless you had good reason?!

Reply:
Fixed and updated the test coverage for this item.